RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -598,54 +598,79 @@ void auto_scepterSkills()
 		}
-		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && pathHasFamiliar())
+		else if((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small())
 		{
-			familiar hundred_fam = to_familiar(get_property("auto_100familiar"));
-			if (((in_avantGuard() && in_hardcore()) || (hundred_fam != $familiar[none] && (isAttackFamiliar(hundred_fam) || hundred_fam.block))) && have_familiar(findRockFamiliarInTerrarium()))
+			if(!is100FamRun())
 			{
-				use_familiar(findRockFamiliarInTerrarium());
-				use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment to lower weight of attack familiar or Burly bodyguard (Avant Guard) for Gremlins
+				use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
 			}
-			else if((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small())
+			else
 			{
-				if(!is100FamRun())
-				{
-					use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
-				}
-				else
-				{
-					use_familiar(hundred_fam); // assuming non-rock familiar
-				}
-				use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+				use_familiar(hundred_fam); // assuming non-rock familiar
 			}
+			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
 		}
 	}
-	//Day 2+ skills
-	if(my_daycount() >= 2)
+	//see how much mana cost reduction we can get (up to 3mp)
+	maximize("-mana cost", true);
+	int manaCostMaximize = numeric_modifier("Generated:_spec", "Mana Cost");
+	if(manaCostMaximize < 3 && canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean() && get_property("_augSkillsCast").to_int()< 5)
+	{
+		use_skill($skill[Aug. 30th: Beach Day!]); //For -MP (and Rollover Adventures)
+	}
+}
+
+void auto_scepterRollover()
+{
+	//We don't want the baywatch if our accessory slots are already filled with > 7 adventure items or we if one of the slots is the counterclockwise watch
+	boolean noWatch = ((numeric_modifier(equipped_item($slot[acc1]),"Adventures") >= 7 &&
+	numeric_modifier(equipped_item($slot[acc2]),"Adventures") >= 7 &&
+	numeric_modifier(equipped_item($slot[acc3]),"Adventures") >= 7) ||
+		((is_watch(equipped_item($slot[acc1])) && numeric_modifier(equipped_item($slot[acc1]),"Adventures") >= 7) ||
+		(is_watch(equipped_item($slot[acc2])) && numeric_modifier(equipped_item($slot[acc2]),"Adventures") >= 7) ||
+		(is_watch(equipped_item($slot[acc3])) && numeric_modifier(equipped_item($slot[acc3]),"Adventures") >= 7)));
+	if(!noWatch && canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean() && get_property("_augSkillsCast").to_int()< 5)
+	{
+		use_skill($skill[Aug. 30th: Beach Day!]); //For Rollover adventures (and -MP)
+		equipRollover(true);
+	}
+	//Get mainstats
+	if(get_property("_augSkillsCast").to_int()< 5 && disregardInstantKarma())
```

Comment:
```suggestion
	if(get_property("_augSkillsCast").to_int()< 5 && my_level() < 13)
```
we only want to use a limited resource to level if we're not done with leveling yet.